Library/Homebrew/livecheck.rb

```diff
@@ -40,6 +41,16 @@ def skip?
     @skip
   end
 
+  # Sets the strategy instance variable to the argument given or returns the
+  # strategy instance variable when no argument is given.
+  # @param symbol [Symbol] corresponding symbol for desired LivecheckStrategy
+  # @return [Symbol, nil]
+  def strategy(symbol = nil)
+    return @strategy if symbol.nil?
+
+    @strategy = symbol if symbol.is_a?(Symbol)
```

Comment:
I think my main question is if we should be guarding against a non-`Symbol` argument here. The other DSL methods simply accept whatever you give them.

Reply:
I think ideally all will `raise` on invalid input (including this one).

Reply:
Sounds good to me. How do you feel about something like this?

```suggestion
    raise TypeError, "Livecheck#strategy expects a Symbol" unless symbol.is_a?(Symbol)

    @strategy = symbol
```

I was planning to add additional `@param` and `@return` comments to the other methods, so I can bring the other methods in line when I create that PR unless you would prefer it to be done in this one.

Reply:
I think you could do

```ruby
case symbol
when Symbol
  @strategy = symbol
when nil
  @strategy
else
  raise ...
end
```